divvy/cloudfilebrowser.scss

```diff
@@ -527,36 +525,37 @@ the License.
         content: '\f15c';
         display: inline;
         font-family: 'FontAwesome';
-        font-size: 13px;
+        font-size: 1.5rem;
```

Comment:
I cannot tell for sure, but does this only apply the font-size to a very specific thing? Ideally this font size would be applied to the whole modal.